product-web-app/app.ts

```diff
@@ -61,15 +69,11 @@ new JobSpecificationController().init(app);
 
 app.listen(3000, () => {
   // eslint-disable-next-line no-console
-  
+  console.log('Server started on port 3000');
 });
 
 app.get('/', async (req, res) => {
-  // if (!req.session.token || req.session.token.length === 0) {
-  //   res.redirect('auth/login');
-  // } else {
-    res.redirect('/job-roles');
-  // }
+  res.redirect('/home');
```

Comment:
app should default to /job-roles and if not authorised then redirect to /login (but that should be  in middleware)